docs/installing_daemons.md

```diff
@@ -17,11 +17,11 @@ use the pip-based install if you have a good reason and know how to
 configure your web service manually:
 
 ```bash
-pip install rucio
+$ pip install rucio
```

Comment:
while I am not strictly against adding the shell prefix, it does make these things harder to copy-paste since you cannot use the copy button for the generated code area. If there is no way to make the prefix just a decoration, please revert these.

Reply:
On the same lines, I think so we should remove all the Shell Prefixes as they are copied whenever the copy button is clicked... I will check if this behaviour can be changed 🤔